### PR TITLE
Specify GCC/QEMU path and check python version

### PR DESCRIPTION
**Pull request by lecopzer**
1. Check python version should >= 3.5 to import `runner`
2. Avoid hardcode of GCC/QEMU path

### CODE REVIEW

File: tests/__main__.py [hunk 1]
```diff
@@ -1,3 +1,7 @@
+import sys
+if sys.version_info < (3, 5):
+    print ("Only support python version >= 3.5")
+    exit(-1)
```
Comment (mlouielu) [inline tests/__main__.py]: This should be `exit(1)`, ref: [exit code](http://tldp.org/LDP/abs/html/exitcodes.html)

And also need a newline after that.

Comment (lecopzer) [reply tests/__main__.py]: Done

**Comment by jserv** [comment]
@lecopzer, Travis-CI complains as following:
```
pip 9.0.1 from /home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages (python 3.5)
Could not locate requirements.txt. Override the install: key in your .travis.yml to install dependencies.
Please override the script: key in your .travis.yml to run tests.
```

Can you please send another pull request for fixing?

**Comment by lecopzer** [comment]
OK, I'll make Travis work well

**Pull request merged**